apps/designer/app/canvas/shared/instance.ts

```diff
@@ -189,37 +183,16 @@ export const useDeleteInstance = () => {
 
 export const usePublishSelectedInstanceData = () => {
   const [instance] = useSelectedInstance();
-  const [selectedElement] = useSelectedElement();
-  const [allUserProps] = useAllUserProps();
-
-  // trigger style recomputing every time
-  // new updates for this node are sent from designer
-  const [styleKey, recomputeStyles] = useReducer((d) => d + 1, 0);
-  useSubscribe("updateStyle", ({ id }) => {
-    if (id === instance?.id) {
-      recomputeStyles();
-    }
-  });
 
   useEffect(() => {
     // Unselects the instance by `undefined`
-    let payload: undefined | SelectedInstanceData;
-    if (instance !== undefined) {
-      const props = allUserProps[instance.id];
-      const browserStyle = getBrowserStyle(selectedElement);
-      payload = {
-        id: instance.id,
-        component: instance.component,
-        cssRules: instance.cssRules,
-        browserStyle,
-        props,
-      };
+    if (instance === undefined) {
```

Comment:
having that usePublishSelectedInstanceData is used and also have publish I would give a link in comment on Connector code